shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -758,7 +759,7 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
   public String getUsage() {
     return "cp "
         + "[-R] "
```

Comment:
Is there a reason we use a capital R vs lowercase? Or is it possible to use both? It doesn't seem consistent across many standard bash commands but I'm a fan of not always having to press my shift key to type in options.

Some examples
- [cp](http://man7.org/linux/man-pages/man1/cp.1.html) can use `-r`, `-R`, or `--recursive`
- [rm](http://man7.org/linux/man-pages/man1/rm.1.html) is the same as cp
- [scp](https://linux.die.net/man/1/scp) only support `-r`
- [ls](http://man7.org/linux/man-pages/man1/ls.1.html) `-R` or `--recursive` (`-r` is taken by `--reverse` sorting flag)
- [chmod](https://linux.die.net/man/1/chmod) `-R` or `--recursive`. There is no conflict with another `-r` option. It just doesn't take the lowercase flag.

Reply:
agree we should keep our shell CLI consistent. Can you create a separate PR on recursive option? I think `-r` or `--recursive`  can work.  and might need to change`fs ls -r` to `fs ls --reverse`. But I am cool with that

Reply:
Opened https://github.com/Alluxio/alluxio/issues/8498